dvc/repo/__init__.py

```diff
@@ -351,16 +350,6 @@ def fs(self, fs: "FileSystem"):
         # fs.
         self._reset()
 
-    @property
-    def subrepo_relpath(self) -> str:
```

Comment:
[F] kept this the same as in https://github.com/iterative/dvc/pull/10291 because it doesn't fit nicely on `Repo`